Modules/Filtering/LabelMap/wrapping/test/LabelImageToStatisticsLabelMapFilterTest2.py

```diff
@@ -0,0 +1,70 @@
+# ==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ==========================================================================*/
+
+#
+#  Test for LabelImageToStatisticsLabelMapFilter
```

Comment:
I think this comment is not useful, and should be removed.

Reply:
Done.  I also added blank lines and commas where the `black` Python formatter recommended them.